src/bot/events/messageUpdate.js

```diff
@@ -20,7 +20,8 @@ module.exports = {
     }
     if (!oldMessage) return
     if (newMessage.author.bot && !global.bot.global.guildSettingsCache[newMessage.channel.guild.id].isLogBots()) return
-    if ((newMessage.content === oldMessage.content) && (newMessage.attachments.length === oldMessage.attachment_b64.split("|").filter(Boolean).length)) return // content/attachments didn't change so don't process
+    // NB: Messages are stored with markdown escaped, so when comparing content we need to escape the new content
+    if ((escape(newMessage.content) === oldMessage.content) && (newMessage.attachments.length === oldMessage.attachment_b64.split("|").filter(Boolean).length)) return // content/attachments didn't change so don't process
```

Comment:
eh.. shouldn't it be 
```js
escape(newMessage.content.replace(/~/g, '\\~'), ['angle brackets'])
```
or like the one from later in the code? 
https://github.com/TizzySaurus/logger/blob/development/src/bot/events/messageUpdate.js#L64

Also it looks like for every message update it doesn't store the escaped markdown version. (even though on `MESSAGE_CREATE` it does)